frontend/workflows/projectSelector/src/types.tsx

```diff
@@ -117,12 +117,16 @@ export interface TimeData {
   [eventsKey: string]: EventData;
 }
 
+export interface TimeseriesPoint extends IClutch.timeseries.v1.IPoint {
```

Comment:
why not add this field to the proto?

Reply:
we can, will update. i guess i saw the proto serving more for backend transformations of the API response

Reply:
updated